### ISSUE TITLE
uuid not generated if property name is other than "_id"

### ISSUE THREAD

**Issue by YannisMarios** [json-schema-faker/json-schema-faker#280]
Hello,

My schema: role.schema.lib.json
```
{
	"$schema": "http://json-schema.org/draft-04/schema#",
	"title": "Role",
	"type": "object",
	"properties": {
		"_id": {
			"type": "string",
			"pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
		},
		"name": {
			"type": "string",
			"maxLength": 48
		},
		"createdAt": { "type": "string", "format": "date-time" }
	},
	"required": ["name", "createdAt"],
	"additionalProperties": false
}
```

My class:

```
import jsf from 'json-schema-faker';
import Schema from '../../schemas/users/role.schema.lib';

class RoleFake {
	async getFake() {
		return await jsf.resolve(Schema);
	}
}

export default RoleFake;
```

calling class above from test:

```
let fake = new RoleFake();
console.log(await fake.getFake());
```

gives 

```
{ name: 'consectet',
  createdAt: '1981-12-03T08:00:11.630Z',
  _id: 'eA1cbc88-dDfD-1baf-2E33-CD1EbE12E98D' }
```

but if I change _id to id in my schema then

```
{ name: 'magna adipisicing in ea',
  createdAt: '1967-02-14T18:08:22.312Z' }
```

why is this happening?

I want to use your to test a service that is using RethinkDB.

Unlike MongDB, RethinkDB is using id instead _id.

Thank you in advance

**Comment by ducin** [comment]
@YannisMarios would you mind editing your issue and wrap code snippets with backticks, like `this one` and

```
that one
```

too?

It'd make it a lot easier to grab your issue :) thanks in advance!

**Comment by YannisMarios** [comment]
sure..no problem I will do that right now

**Comment by YannisMarios** [comment]
I tried but it doesn't work


have a look at the backticks

**Comment by ducin** [comment]
@YannisMarios I've done that for you

**Comment by YannisMarios** [comment]
thank you

**Comment by pateketrueke** [comment]
Sorry, this is related with #273 and it'll be fixed on `0.5.0-rc4` soon!

**Comment by YannisMarios** [comment]
Hi pateketrueke

Any idea when soon is? more or less? :-)

**Comment by pateketrueke** [comment]
I'm planning working on this today, just now I'm reviewing all related issues.

**Comment by YannisMarios** [comment]
ok...thank you very much

**Comment by pateketrueke** [comment]
If everything goes fine, I'll release a patch also today.

**Comment by YannisMarios** [comment]
Temporary solution until it gets fixed:
`import jsf from 'json-schema-faker';
import Schema from '../../schemas/users/role.schema.lib';

function fix_key(key) { return key.replace(/^_id/, 'id'); }

class RoleFake {
	async getFake() {
		let role = await jsf.resolve(Schema);
		Object.assign({},...Object.keys(role).map(key => ({[fix_key(key)]: role[key]})));
		return role;
	}
}

export default RoleFake;`

**Comment by ducin** [comment]
@YannisMarios please do following using backticks:
'''
code code
code 
'''

**Comment by YannisMarios** [comment]
Temporary solution until it gets fixed
```
import jsf from 'json-schema-faker';
import Schema from '../../schemas/users/role.schema.lib';

function fix_key(key) { return key.replace(/^_id/, 'id'); }

class RoleFake {
	async getFake() {
		let role = await jsf.resolve(Schema);
		Object.assign({},...Object.keys(role).map(key => ({[fix_key(key)]: role[key]})));
		return role;
	}
}

export default RoleFake;
```

**Comment by pateketrueke** [comment]
Fixed and released `0.5.0-rc4` 👍 

Btw; you should add `id` as `required` then, otherwise it may appear... randomly.
